eos/constraint_TEST.cc

```diff
@@ -965,6 +965,51 @@ class ConstraintDeserializationTest :
                 TEST_CHECK_NEARLY_EQUAL(llh(), -4.597666149, 1e-8);
             }
             // }}}
+
+            // {{{ MultivariateGaussian(Covariance) (value; sub-sample)
+            {
+                static const std::string input(
+                    "type: MultivariateGaussian(Covariance)\n"
+                    "dim: 3\n"
+                    "observables:\n"
+                    "  - mass::b(MSbar)\n"
+                    "  - mass::c\n"
+                    "  - mass::K_d\n"
+                    "kinematics:\n"
+                    "  - {}\n"
+                    "  - {}\n"
+                    "  - {}\n"
+                    "options:\n"
+                    "  - {}\n"
+                    "  - {}\n"
+                    "  - {}\n"
+                    "means: [4.3, 1.1, 6.2]\n"
+                    "covariance:\n"
+                    "  - [0.0100, 0.0030, 0.0060]\n"
+                    "  - [0.0030, 0.0025, 0.0015]\n"
+                    "  - [0.0060, 0.0015, 0.0105]\n"
+                    "dof: 3"
+                );
+
+                YAML::Node node = YAML::Load(input);
+
+                std::shared_ptr<ConstraintEntry> entry(ConstraintEntry::FromYAML("Test::MultivariateGaussian(Covariance)", node));
+                TEST_CHECK(nullptr != entry.get());
+
+                Constraint c = entry->make("Test::MultivariateGaussian(Covariance)", Options{{"begin","0"}, {"end","2"}});
+		std::vector<LogLikelihoodBlockPtr> blocks(c.begin_blocks(), c.end_blocks());
```

Comment:
Again, what happened with the indentations? Did you use tabs?

Reply:
hmm emacs is indenting automatically and displays the result correctly. I don't know why it appears weird on github, I will fix this.